_posts/2019-01-11-Why-is-Apiary-using-CircleCI.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Why is Apiary using CircleCI?
+excerpt: We started using CircleCI in 2012. This has been our experience with it in the last 7 years.
+layout: post
+date: 2019-01-31 00:00:00 +0100
+author: abtris
+published: true
+---
+
+Apiary is using multiple Continuous Integration Engines (CIE). Our testing tool [Dredd](https://dredd.org/en/latest/), for example, uses [TravisCI](https://travis-ci.org/) to support multiple versions of Node.JS, and [AppVeyor](https://www.appveyor.com/) to support Windows.
```

Comment:
Since the tests don't pass on Circle out of the box, I wanted to port Dredd Transactions first and then to port the Dredd tests, slowly debugging the problems. I might get to the first phase soon, but the second is tedious and low priority at this moment.